pkg/kodo/cmd/deploy.go

```diff
@@ -16,7 +23,25 @@ type DeploymentVariables struct { //New struct for deployment creation variables
 	Port     int32
 }
 
-func Deploy(deployVar *DeploymentVariables, envVar *EnvironmentVariables) error {
+type DeploymentIdentifiers struct { //New struct to hold unique identifiers for deployment/service/route
+	DeploymentName string
```

Comment:
Do we need separate names/variables for every resource?

Reply:
I thought it would be better practice and it might be confusing otherwise
should I make them the same and create a new PR?

Reply:
Generate one unique identifier and use it for all objects.

What do you think?

Reply:
sounds good

Reply:
And to be clear, reviews are to be handled in the same PR. Just push to the same branch and I will show up here.